docs/configuration/transit-provider.md

```diff
@@ -1,49 +1,118 @@
 
-### Steps
+This work can begin once the transit provider has a contract in place with Littlepay.
+
+- Cal-ITP uses the transit provider's Littlepay merchant ID to create a customer group in the Littlepay QA environment for each type of eligibility (e.g. senior).
+  - _Typically performed by transit provider's Account Manager_
+  - For each group that's created, a group ID will be returned and should be set as the `group_id` on a new `LittlepayGroup` in the Benefits database. (See [Configuration data](../data/) for more on loading the database.)
+- Cal-ITP requests and receives Littlepay Back Office API access (for both PROD and QA) for the new transit provider.
+  - _Typically requested by a developer via email to Littlepay_
+
+### Basic transit provider data configuration
+
+--8<-- "inc/add-transitagency.md"
+--8<-- "inc/add-enrollmentflow.md"
 
-1. Cal-ITP uses the transit provider's Littlepay merchant ID to create a customer group in the Littlepay QA environment for each type of eligibility (e.g. senior).
-1. For each group that's created, a group ID will be returned and should be set as the `group_id` on a new `EnrollmentFlow` in the Benefits database. (See [Configuration data](../data/) for more on loading the database.)
-1. Cal-ITP creates a new `EnrollmentFlow` in the database for each supported eligibility type. This will require configuration for either [API](https://docs.calitp.org/eligibility-api/specification/)-based verification or verification through an [OAuth Open ID Connect claims provider](../oauth/) (e.g. sandbox Login.gov) -- either way, this resource should be meant for testing.
-1. Cal-ITP creates a new `TransitAgency` in the database and associates it with the new `EnrollmentFlow`s as well as the existing Littlepay `TransitProcessor`.
 
 ## Configuration for production validation
 
-For production validation, both a customer group and discount product are needed. The customer group used here is a temporary one for testing only. Production validation is done against the Benefits **test environment** to avoid disruption of the production environment.
+**Production validation** is the process of doing a end-to-end test of enrolling a real person's card through the Benefits app and using it to ride with a discounted fare. The word "production" here refers to **Littlepay's production environment** (which must be used to take a ride in real life), but the Benefits application's test environment is used for the enrollment process to avoid disruption of the Benefits production environment.
 
-### Steps
+For production validation, both a customer group and discount product are needed. The customer group used here is a temporary one for testing only.
 
 1. Transit provider staff creates the discount product in production Littlepay (if it does not already exist).
 1. Transit provider staff takes a screenshot of the discount product in the Merchant Portal, making sure the browser URL is visible, and sends that to Cal-ITP.
```

Comment:
This is outdated in a number of ways.

- This was written back when the transit provider staff used something called a "Merchant Portal" to create the products, and the browser URL was the only place that the product ID was exposed. I'm not sure why we decided to ask them to take a screenshot rather than just copy-pasting the URL for us :joy: but nonetheless, these days, they're using something called the Control Portal, which exposes the IDs more clearly.

- When this was written, Cal-ITP engineers needed the product IDs to link the products to the groups. These days, account managers are working with agency staff directly to link products and groups, so engineers are no longer involved in this step nor do we need the product IDs.

Reply:
Caveat: engineers would need the product IDs if we were to go with the approach of engineers managing test groups in production for production validation, as mentioned in https://github.com/cal-itp/benefits/pull/3233#discussion_r2457111176